src/transformers/models/longformer/modeling_longformer.py

```diff
@@ -1407,6 +1407,28 @@ def _set_gradient_checkpointing(self, module, value=False):
             module.gradient_checkpointing = value
 
 
+LONGFORMER_GENERATION_DOCSTRING = r"""
```

Comment:
This example looks super cool - could you replace it with what's currently written for `LongformerForMaskedLM` ?

Reply:
Agree with @patrickvonplaten 

A few more comments:
- No more need for `LONGFORMER_GENERATION_DOCSTRING`, but moving the docstring example to `LongformerForMaskedLM` directly

- I think it would be great to add a comment before the line containing ` * 300`, say `Let's try a very long input.`